prettyprinter/src-text/Data/Text/Lazy/Builder.hs

```diff
@@ -7,7 +7,24 @@
 
 module Data.Text.Lazy.Builder where
 
-type Builder = String
-fromText = id
-singleton = (:[])
-toLazyText = id
+import Data.String (IsString (..))
+
+newtype Builder =  Builder (String -> String)
```

Comment:
Why is this necessary?

When @Bodigrim proposed the `text` flag, my expectation was that the `-f-text` variant would not be required to perform well.

I also thought that `-f-text` users would stick to `renderString` and not require the other renderers. Do you want to use `renderLazy` in `o-a`? How come?